python_modules/dagster-webserver/dagster_webserver/cli.py

```diff
@@ -26,35 +27,32 @@
 from .version import __version__
 
 
-def create_dagit_cli():
-    return dagit
+def create_dagster_webserver_cli():
+    return dagster_webserver
 
 
-DEFAULT_DAGIT_HOST = "127.0.0.1"
-DEFAULT_DAGIT_PORT = 3000
+DEFAULT_WEBSERVER_HOST = "127.0.0.1"
+DEFAULT_WEBSERVER_PORT = 3000
```

Comment:
nit: "webserver" feels redundant here

Reply:
I would agree if these were module-local but they are imported elsewhere so I think "WEBSERVER" is a useful qualifier.